website_legal_age_notice/__manifest__.py

```diff
@@ -22,7 +22,7 @@
     'name': 'Legal age notice',
     'category': 'website',
     'summary': 'Show legal age notice according to law',
-    'version': '8.0.0.1',
+    'version': '12.0.0.1',
```

Comment:
We're changing all version numbers to 5 digits. So start with this one...
```suggestion
    'version': '12.0.1.0.0',
```